### PR TITLE
Use username also to log in

### PR DESCRIPTION
**Pull request by Akarshit**
Fix for #247

### CODE REVIEW

File: client/modules/User/components/LoginForm.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ function LoginForm(props) {
   return (
     <form className="form" onSubmit={handleSubmit(props.validateAndLoginUser.bind(this, props.previousPath))}>
       <p className="form__field">
-        <label htmlFor="email" className="form__label">Email</label>
+        <label htmlFor="email" className="form__label">Email/Username</label>
```
Comment (catarak) [inline client/modules/User/components/LoginForm.jsx]: To me this is more clear as "Email or Username", to me "Email/Username" implies that they are one in the same

File: client/modules/User/components/LoginForm.jsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ function LoginForm(props) {
...
         <input
           className="form__input"
           aria-label="email"
```
Comment (catarak) [inline client/modules/User/components/LoginForm.jsx]: could you update this to
```
aria-label="email or username"
```

**Comment by catarak** [comment]
ty this is great 👼 just have a couple tiny changes!

File: client/modules/User/components/LoginForm.jsx (hunk 2 above)
Comment (Akarshit) [inline client/modules/User/components/LoginForm.jsx]: Do we need this even after changing `Email/Username` to `Email or Username`?

**Pull request merged**